_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -111,6 +113,22 @@ WAI-ARIA states and properties with empty value, specified as an empty string, a
 <div role="combobox" aria-controls="">My combobox</div>
 ```
 
+#### Passed Example 9
+
+This `aside` element has an [explicit role][] of `none`. However, the [global][] [property][] `aria-label` is specified. Thus it has a [semantic role][] of `complementary` due to [Presentational Roles Conflict Resolution][]. The `aria-expanded` [state][] is [inherited][] for the `complementary` role.
+
+```html
+<aside role="none" aria-label="About ACT rules" aria-expanded="true">ACT rules are cool!</aside>
+```
+
+#### Passed Example 10
+
+This `aside` element has an [explicit role][] of `none`. However, it is [focusable][] due to the `tabindex` attribute (even if it is excluded from sequential focus order). Thus it has a [semantic role][] of `complementary` due to [Presentational Roles Conflict Resolution][]. The `aria-expanded` [state][] is [supported][] for the `complementary` role.
+
+```html
+<aside role="none" tabindex="-1" aria-expanded="true">ACT rules are cool!</aside>
```

Comment:
The behavior of AT here is not consistent. JAWS does not report the expanded state. NVDA only reports it in Firefox.